myreactapp/src/Newfan.js

```diff
@@ -0,0 +1,24 @@
+import { render } from "@testing-library/react";
+
+class Newfan extends Component{
+    constructor(props){
+      super(props)
+      this.state = {fanStatus: ''};
+  
+    }
+  
+
+  changeState = () => {
+    this.setState({fanStatus: 'I AM A FAN!'})
+  }
+
+  render() {
+      return(
+
+        <div>
+            <h3>Are you a fan of the work?</h3>
+            <button onClick = {this.changeState}>CLICK HERE!</button>
+        </div>
+      )
+  }
+}
```

Comment:
Don't forget to include the export call. It should go on a new line after line 24.  With out adding `export default Newfan`, you aren't able to import it from your app.js file.